src/views/clusteroverview/SettingsTab/ClusterTab/components/GuestManagmentSection/GuestSystemLogsAccess/GuestSystemLogsAccess.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { FC, useState } from 'react';
+
+import HyperConvergedModel from '@kubevirt-ui/kubevirt-api/console/models/HyperConvergedModel';
+import HelpTextIcon from '@kubevirt-utils/components/HelpTextIcon/HelpTextIcon';
+import { HyperConverged } from '@kubevirt-utils/hooks/useHyperConvergeConfiguration';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { k8sPatch } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  Alert,
+  AlertVariant,
+  PopoverPosition,
+  Split,
+  SplitItem,
+  Switch,
+} from '@patternfly/react-core';
+
+import './guest-system-logs-access.scss';
+
+type GuestSystemLogsAccessProps = {
+  hyperConvergeConfiguration: [hyperConvergeConfig: HyperConverged, loaded: boolean, error: Error];
+};
+
+const GuestSystemLogsAccess: FC<GuestSystemLogsAccessProps> = ({ hyperConvergeConfiguration }) => {
+  const { t } = useKubevirtTranslation();
+  const [hyperConverge, hyperLoaded] = hyperConvergeConfiguration;
+  const disableSerialConsoleLog =
+    hyperConverge?.spec?.virtualMachineOptions?.disableSerialConsoleLog;
+
+  const [error, setError] = useState<Error>(null);
+  const [isChecked, setIsChecked] = useState<boolean>(
+    disableSerialConsoleLog === 'true' || disableSerialConsoleLog === undefined,
+  );
+
+  const onChange = async (checked: boolean) => {
+    try {
+      await k8sPatch<HyperConverged>({
+        data: [
+          {
+            op: 'replace',
+            path: `/spec/virtualMachineOptions/disableSerialConsoleLog`,
+            value: checked,
+          },
+        ],
+        model: HyperConvergedModel,
+        resource: hyperConverge,
+      });
+      setIsChecked(checked);
+    } catch (e) {
```

Comment:
I'd use here `err` at least, to make it more visually clear that it is about the error. Because `e` could also feel like event (even when in a different situation, but still.. , if we want to name the variables nicely and clearly)

Reply:
+1

Reply:
sure will change to error